src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -730,3 +730,62 @@ def computer_config_show(computer, user, defaults, as_option_string):
             else:
                 table.append((f'* {name}', '-'))
         echo_tabulate(table, tablefmt='plain')
+
+
+@verdi_computer.group('export', cls=LazyConfigureGroup)
+def computer_export():
+    """Exports the Authinfo details for a computer (and user)."""
+
+
+@computer_export.command('setup')
+@arguments.COMPUTER()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False))
+@with_dbenv()
+def computer_export_setup(computer, output_file):
+    """Export computer setup to a yaml file."""
+    import yaml
+
+    computer_setup_data = {
+        'label': computer.label,
+        'description': computer.description,
+        'hostname': computer.hostname,
+        'transport': computer.transport_type,
+        'scheduler': computer.scheduler_type,
+        'work_dir': computer.get_workdir(),
+        'shebang': computer.get_shebang(),
+        'mpirun_command': ' '.join(computer.get_mpirun_command()),
+        'mpiprocs_per_machine': computer.get_default_mpiprocs_per_machine(),
+        'default_memory_per_machine': computer.get_default_memory_per_machine(),
+        'prepend_text': computer.get_prepend_text(),
+        'append_text': computer.get_append_text(),
+        'use_double_quotes': computer.get_use_double_quotes(),
+    }
+    try:
+        with open(output_file, 'w', encoding='utf-8') as yfhandle:
+            yaml.dump(computer_setup_data, yfhandle)
+        echo.echo_success(f"Computer<{computer.pk}> {computer.label} setup exported to file '{output_file}'.")
+    except Exception:
+        raise
+
+
+@computer_export.command('config')
+@arguments.COMPUTER()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False))
+@with_dbenv()
+def computer_export_config(computer, output_file):
+    """Export computer configuration to a yaml file."""
+    import yaml
+
+    if not computer.is_configured:
+        echo.echo_error(
+            f'Computer<{computer.pk}> {computer.label} config cannot be exported,'
+            'because computer has not been configured yet.'
+        )
+    else:
```

Comment:
```suggestion
    if not computer.is_configured:
        echo.echo_critical(
            f'Computer<{computer.pk}> {computer.label} config cannot be exported,'
            'because computer has not been configured yet.'
        )
```
`echo_critical` will exit the command and return a non-zero exit code automatically. This is useful for when the command is called in scripts as it can then determine if it was successful.

Also, this now allows to get rid of the `else` and you can dedent the rest of the command, making it easier to read.

Reply:
done